portal-api/src/v1/controllers/deal-status/send-status-update-emails.js

```diff
@@ -1,12 +1,11 @@
 const sendEmail = require('../../email');
 
-const sendEmailsToOwningBanks = async (templateId, emailVariables, owningBankEmails) => {
+const sendEmailsToOwningBanks = async (templateId, emailVariables, owningBankEmails) =>
```

Comment:
So there is a slight difference between these two definitions: the one with `{}` will return `Promise<void>`, whilst the other will return `Promise<unknown[]>`. Are we sure it's okay for this function to now be returning an array?

Reply:
absolutely not, great catch